### ISSUE TITLE
Accessibility: Default Collection thumbnail have no alt text

### ISSUE THREAD

**Issue by flutterflies** [uclibs/scholar_uc#1688]
### Descriptive summary

The collection thumbnail shown when the creator does not upload a thumbnail has no alt text for screen readers
